From 7a0f939a37d6a624edd4c3cad3bb421751cc5299 Mon Sep 17 00:00:00 2001
From: Nishidha Panpaliya <npanpa23@in.ibm.com>
Date: Thu, 25 Mar 2021 08:40:54 -0400
Subject: [PATCH] Trial change

---
 configure.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/configure.py b/configure.py
index 4772a55..3f2567e 100644
--- a/configure.py
+++ b/configure.py
@@ -118,8 +118,8 @@ def create_build_configuration():
         write("build:windows --host_copt=/experimental:preprocessor")
         write("build:windows --copt=/arch=AVX")
 
-    if is_macos() or is_linux():
-        write("build --copt=-mavx")
+    #if is_macos() or is_linux():
+    #    write("build --copt=-mavx")
 
     if os.getenv("TF_NEED_CUDA", "0") == "1":
         print("> Building GPU & CPU ops")
-- 
1.8.3.1

