From fae7aadb6037e81e06e910cd39733b7465f69e24 Mon Sep 17 00:00:00 2001
From: Archana Shinde1 <archana.shinde2504@gmail.com>
Date: Fri, 22 Dec 2023 10:59:21 +0000
Subject: [PATCH] Fix pip check failure

---
 requirements.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/requirements.txt b/requirements.txt
index d060a47..0e80127 100644
--- a/requirements.txt
+++ b/requirements.txt
@@ -1,2 +1,2 @@
-typeguard>=2.7,<3.0.0
+typeguard>=2.7,<=3.0.2
 packaging
-- 
2.40.1

