diff --git a/configure.py b/configure.py
index 5126f6d..71c5926 100644
--- a/configure.py
+++ b/configure.py
@@ -141,7 +141,7 @@ def create_build_configuration():
         write("build:windows --host_cxxopt=/std:c++14")
 
     if is_macos() or is_linux():
-        if not is_linux_ppc64le() and not is_linux_arm() and not is_linux_aarch64():
+        if not is_linux_ppc64le() and not is_linux_s390x() and not is_linux_arm() and not is_linux_aarch64():
             write("build --copt=-mavx")
         write("build --cxxopt=-std=c++14")
         write("build --host_cxxopt=-std=c++14")

